exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricTranslator.java

```diff
@@ -104,28 +114,69 @@ static MetricDescriptor mapMetricDescriptor(
         .forEach((key, value) -> builder.addLabels(mapAttribute(key, value)));
 
     MetricDataType metricType = metric.getType();
-    if (GAUGE_TYPES.contains(metricType)) {
-      builder.setMetricKind(MetricDescriptor.MetricKind.GAUGE);
-    } else if (CUMULATIVE_TYPES.contains(metricType)) {
-      builder.setMetricKind(MetricDescriptor.MetricKind.CUMULATIVE);
-    } else {
-      logger.error(
-          "Metric type {} not supported. Only gauge and cumulative types are supported.",
-          metricType);
-      return null;
-    }
-    if (LONG_TYPES.contains(metricType)) {
-      builder.setValueType(MetricDescriptor.ValueType.INT64);
-    } else if (DOUBLE_TYPES.contains(metricType)) {
-      builder.setValueType(MetricDescriptor.ValueType.DOUBLE);
-    } else {
-      logger.error(
-          "Metric type {} not supported. Only long and double types are supported.", metricType);
-      return null;
+    switch (metricType) {
+      case LONG_GAUGE:
+        builder.setMetricKind(MetricDescriptor.MetricKind.GAUGE);
+        builder.setValueType(MetricDescriptor.ValueType.INT64);
+        break;
+      case DOUBLE_GAUGE:
+        builder.setMetricKind(MetricDescriptor.MetricKind.GAUGE);
+        builder.setValueType(MetricDescriptor.ValueType.DOUBLE);
+        break;
+      case LONG_SUM:
+        builder.setValueType(MetricDescriptor.ValueType.INT64);
+        fillSumType(metric.getLongSumData(), builder);
+        break;
+      case DOUBLE_SUM:
+        builder.setValueType(MetricDescriptor.ValueType.DOUBLE);
+        fillSumType(metric.getDoubleSumData(), builder);
+        break;
+      case HISTOGRAM:
+        fillHistogramType(metric.getDoubleHistogramData(), builder);
+        break;
+      default:
+        logger.error(
+            "Metric type {} not supported. Only gauge and cumulative types are supported.",
+            metricType);
+        return null;
     }
     return builder.build();
   }
 
+  private static void fillHistogramType(
+      DoubleHistogramData histogram, MetricDescriptor.Builder builder) {
+    builder.setValueType(MetricDescriptor.ValueType.DISTRIBUTION);
+    switch (histogram.getAggregationTemporality()) {
+      case DELTA:
+        builder.setMetricKind(MetricDescriptor.MetricKind.DELTA);
```

Comment:
I don't think delta is supported for custom metrics at all right now (see [here](https://cloud.google.com/monitoring/custom-metrics/creating-metrics#:~:text=metricKind%3A%20The%20default%20metric%20kind%20is%20GAUGE%2C%20but%20if%20you%20specify%20the%20metricKind%20parameter%20of%20the%20TimeSeries%20object%2C%20then%20the%20new%20metric%20has%20that%20kind.%20You%20can%20specify%20only%20GAUGE%20and%20CUMULATIVE%20kinds.), maybe you're aware of this already, and just want the errors to bubble up anyway) 

cc @kjordy

Reply:
Ok, these are now dropped.  We can fix this going forward once we finish our metric-export specification for how things map.